python/cudf/cudf/tests/test_dataframe_copy.py

```diff
@@ -131,27 +131,6 @@ def test_cudf_dataframe_copy_then_insert(copy_fn, ncols, data_type):
     assert not copy_df.to_string().split() == df.to_string().split()
 
 
-@cuda.jit
-def group_mean(data, segments, output):
-    i = cuda.grid(1)
-    if i < segments.size:
-        s = segments[i]
-        e = segments[i + 1] if (i + 1) < segments.size else data.size
-        # mean calculation
-        carry = 0.0
-        n = e - s
-        for j in range(s, e):
-            carry += data[j]
-        output[i] = carry / n
-
-
-@cuda.jit
-def add_one(data):
-    i = cuda.grid(1)
-    if i == 1:
-        data[i] = data[i] + 1.0
-
-
 def test_kernel_deep_copy():
```

Comment:
The "kernel" name here is no longer accurate, I think.
```suggestion
def test_deep_copy_write_in_place():
```

Reply:
Same for the shallow copy test below.